### PR TITLE
Add Tips tab

### PR DESCRIPTION
**Pull request by howardbaek**
This PR corresponds to https://github.com/FredHutch/loqui/issues/33.

### CODE REVIEW

File: include-tips.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <h3>Tips:</h3>
     <ul>
         <li>Spell out acronyms with hyphens (e.g., AI becomes “aye-eye”).</li>
-        <li>If a word has an unusual pronunciation, consider rephrasing or substituting it with a synonym.</li>
```
[the remark this answers is not in the document]
Comment (carriewright11) [reply include-tips.html]: this has worked for me, so I thought we might want to suggest it

File: include-tips.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+         <li>If a word has an unusual pronunciation, consider rephrasing or substituting it with a synonym.</li>
         <li>For pauses, use 2-3 commas (e.g., “Hello everyone ,,, my name is Howard”).</li>
         <li>Avoid empty slides (ensure every slide has speaker notes).</li>
         <li>Keep the browser open during video rendering.</li>
```
Comment (carriewright11) [inline include-tips.html:7]: I wonder if we can add a note about this where we ask for emails?

File: include-tips.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <h3>Tips:</h3>
     <ul>
         <li>Spell out acronyms with hyphens (e.g., AI becomes “aye-eye”).</li>
-        <li>If a word has an unusual pronunciation, consider rephrasing or substituting it with a synonym.</li>
+         <li>If a word has an unusual pronunciation, consider rephrasing or substituting it with a synonym.</li>
         <li>For pauses, use 2-3 commas (e.g., “Hello everyone ,,, my name is Howard”).</li>
```
Comment (carriewright11) [inline include-tips.html:5]:
```suggestion
<li>For pauses, use 2-3 commas (e.g., “Hello everyone ,,, my name is Howard”), or repeat the image on another slide with more text.</li>
```

File: include-tips.html (hunk 2 above)
Comment (howardbaek) [reply include-tips.html:7]: Yeah good point.

**Comment by howardbaek** [comment]
@carriewright11 I've applied all your suggestions. Do you mind taking a look and approving this PR so I can merge it?

**Review by carriewright11 — APPROVED** [review]

**Pull request merged**